src/index.html

```diff
@@ -56,6 +56,7 @@ <h1>2048</h1>
         <p class="message message-lose hidden">You lose! Restart the game?</p>
         <p class="message message-win hidden">Winner! Congrats! You did it!</p>
         <p class="message message-start">Press "Start" to begin game. Good luck!</p>
+        <p class="message message-over hidden">Game over! Try again.</p>
```

Comment:
The new added line seems to introduce an additional 'Game over' message. Make sure this message is necessary and handled correctly in the game logic. 🤔